pfunk/_io.py

```diff
@@ -17,6 +17,7 @@
 from scipy import stats
 
 import pfunk
+from ._resultsdb import find_test_dates
```

Comment:
Already available via pfunk.find_test_dates

Reply:
Yes, but weirdly, there's _one_ instance of `find_test_dates` (in `generate_report()`) where if I try to use the `pfunk` import then this happens:

```
Generating test report
Traceback (most recent call last):
  File "./funk", line 14, in <module>
    main()
  File "/home/gralee/Projects/pints-team/functional-testing/pfunk/__main__.py", line 515, in main
    args.func(args)
  File "/home/gralee/Projects/pints-team/functional-testing/pfunk/__main__.py", line 195, in generate_report
    pfunk.generate_report(args.database)
  File "/home/gralee/Projects/pints-team/functional-testing/pfunk/_io.py", line 256, in generate_report
    dates = pfunk.find_test_dates(database)
UnboundLocalError: local variable 'pfunk' referenced before assignment
```

I can switch it to `import pfunk as p`, then refer to `p.find_test_dates()`, and that works. But `pfunk.find_test_dates()` does not, only on that one line, out of three uses in the module. :man_shrugging:

Reply:
OK have made separate ticket for this